### DISCUSSION TITLE
Just Completed the blockchain basics

### DISCUSSION THREAD

**Discussion by manishks2901** [Cyfrin/foundry-full-course-cu#1546] [General]
Thank you patrick sir for this amazing blockchain lessons

**Comment by EngrPips** [comment]
Glad you love the lessons

**Comment by ukangaekom** [comment]
Just completed mine as well. It was fun playing around and getting hands dirty with stuffs. Congrats!

**Comment by ARomano79** [comment]
Wait until you get into Solidity and dive even deeper with Foundry.

**Comment by EngrPips** [comment]
That's where things get even more interesting.

**Reply by ARomano79** [reply]
So True!!
